packages/atomic/src/components/common/checkbox.tsx

```diff
@@ -17,6 +17,8 @@ export interface CheckboxProps {
 }
 
 export const Checkbox: FunctionalComponent<CheckboxProps> = (props) => {
+  const isIconPart = props.iconPart !== undefined && props.iconPart !== null;
```

Comment:
You may want to use `isNullOrUndefined` from `@coveo/bueno`
Source:
https://github.com/coveo/ui-kit/blob/7f9b9cb3469e1852765b00d40adca999ba4a0d4c/packages/bueno/src/values/value.ts#L46-L48
E.g in Atomic:
https://github.com/coveo/ui-kit/blob/7f9b9cb3469e1852765b00d40adca999ba4a0d4c/packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx#L1
https://github.com/coveo/ui-kit/blob/7f9b9cb3469e1852765b00d40adca999ba4a0d4c/packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx#L537-L538
And some mind juggling later, here's a silly oneliner implementation without equal sign:
```js
const isNullOrUndefined = (a) => !(!(a ?? true) || a)
```